packages/entities/entities-consumers/src/components/ConsumerList.vue

```diff
@@ -33,22 +34,59 @@
           :disabled="!useActionOutside"
           to="#kong-ui-app-page-header-action-button"
         >
-          <PermissionsWrapper :auth-function="() => canCreate()">
-            <!-- Hide Create button if table is empty -->
+          <div class="button-row">
             <KButton
-              appearance="primary"
-              data-testid="toolbar-add-consumer"
-              :size="useActionOutside ? 'medium' : 'large'"
-              :to="config.consumerGroupId ? undefined : config.createRoute"
-              @click="() => config.consumerGroupId ? handleAddConsumerClick() : undefined"
+              v-if="!showEmptyState && config.app === 'konnect'"
+              appearance="secondary"
+              class="open-learning-hub"
+              data-testid="consumers-learn-more-button"
+              icon
+              @click="$emit('click:learn-more')"
             >
-              <AddIcon />
-              {{ config.consumerGroupId ? t('consumers.actions.add_consumer') : t('consumers.list.toolbar_actions.new_consumer') }}
+              <BookIcon decorative />
             </KButton>
-          </PermissionsWrapper>
+            <PermissionsWrapper :auth-function="() => canCreate()">
+              <!-- Hide Create button if table is empty -->
+              <KButton
+                appearance="primary"
+                data-testid="toolbar-add-consumer"
+                :size="useActionOutside ? 'medium' : 'large'"
+                :to="config.consumerGroupId ? undefined : config.createRoute"
+                @click="() => config.consumerGroupId ? handleAddConsumerClick() : undefined"
```

Comment:
I'm a little confused here by both the `to` and `@click` handlers. If the user clicks the link, they could both fire since the `@click` doesn't have `prevent` etc.

Is there any context as to how this will work?

Reply:
Should I have `@click.prevent` too?

Reply:
Could you just move all of the logic into a click.prevent handler instead of defining both?

Reply:
Looks like this was old code, I had already added a function to do this, so updated to use it here.